README.md

```diff
@@ -64,96 +48,105 @@ _local_ repository uploaded to GitHub. Follow the steps below:
+remote, type `git remote add origin` followed by a space, then paste in the URL.
+It will look something like this:
 
 ```console
-$ git remote add origin your-copied-remote-repository-url
+$ git remote add origin git@github.com:your-github-username/my-new-repo.git
 ```
 
-This sets the remote so you can now **_push_** your code.
+The command above creates a remote called `origin` and points it to your remote
+repo on GitHub.
 
-You can use `git remote -v` (the `-v` is for "verbose") to view the remote(s).
+Let's use `git remote -v` (recall that the `-v` is for "verbose") to verify that
+we successfully set our remote:
 
 ```console
 $ git remote -v
 View existing remotes
-origin  git@github.com:OWNER/REPOSITORY.git (fetch)
-origin  git@github.com:OWNER/REPOSITORY.git (push)
+origin  git@github.com:your-github-username/my-new-repo.git (fetch)
+origin  git@github.com:your-github-username/my-new-repo.git (push)
 ```
 
+We did it! We're now set up to **_push_** our code.
+
 ## Send Code to the Remote Repo with `git push`
 
-Now that we have added a remote repo, we need to send our latest work to the
-remote. We use this command when we want to send some code from the local
-repository to the associated remote repository. Git will push all the local,
-committed work to the remote repository.
+Now that we have added a remote repo, we need to send our latest work to it
+using `git push`. This command will push all the local, committed work to the
+remote repository.
 
 The `git push` command takes two arguments. The first is the name of the remote
 repo. Remember, `origin` is just an alias or "short name" that refers to the
 repository name. You don't actually have to enter the repository name. Instead,
-you can just use `origin`. The second is the name of the remote branch you want
-to send code to. In the example below, we're pushing to our remote repository's
-_default_ branch.
+you can just use `origin`. The second argument is the name of the remote
+**branch** you want to send code to. We're going to push to our remote
+repository's **default branch**.
 
-Depending on your system configuration, your default branch's name might be
-either `main` or `master`. You can find out what the default branch for your
-current repository is by running:
+If you configured Git using the instructions given earlier in the prework, your
+default branch's name will be `main`. You can verify this by running:
 
 ```console
 $ git branch --show-current
 ```
 
-Whatever the branch listed in the terminal is will be your default branch (it
-should be either `main` or `master`).
-
-If your default branch name is `main`, run this command to push your code up
-to GitHub:
-
-```console
-$ git push -u origin main
-```
-
-**Otherwise**, if your default branch name is `master`, run this command to push
-your code up to GitHub:
+Then, to push your code up to GitHub, run this command:
 
 ```console
 $ git push -u origin master
```

Comment:
Since we're assuming students have `main` set up as their default branch, this should be:

```suggestion
$ git push -u origin main
```

Reply:
Oops! Left the wrong one in! :)